desktop/ui/src/main/java/org/datacleaner/windows/DataHubDatastoreDialog.java

```diff
@@ -264,6 +286,11 @@ protected boolean validateForm() {
             return false;
         }
 
+        if (_tenantName == null) {
+            setStatusError("Please test the connection first");
+            return false;
```

Comment:
How about not saving the tenant name at all, but simply requesting it when it is needed in the datacontext?